tests/unit_tests/test_components.py

```diff
@@ -1,3 +1,5 @@
+import __init__  # Running __init__ here enables importing from DOVE and RAVEN
```

Comment:
Does this work if we change `__init__` to `import unit_tests` I think the init file should be ran when we import the module.